site/assets/scss/_component-examples.scss

```diff
@@ -398,8 +398,6 @@
 
 .highlight-toolbar {
   background-color: $gray-100; // Boosted mod: instead of `var(--bd-pre-bg)`
-  border: solid $gray-400; // Boosted mod
-  border-width: 1px 0; // Boosted mod
```

Comment:
It may be too much to remove this line but it's useless since it's overrided by `.border-1`:

```css
.border-1 {
    border-width: 0.0625rem!important;
}
```

But I am currently wondering if we should keep it as a fallback 🤔

Reply:
Some of the examples highlight we have in the docs seem to be broken. + The border-color seems to be not good.